evap/evaluation/models.py

```diff
@@ -388,7 +389,10 @@ def publish(self):
 
     @transition(field=state, source='published', target='reviewed')
     def unpublish(self):
-        pass
+        from evap.results.tools import get_results_cache_key
```

Comment:
PEP8 says you should put the import at the top of the file. Is a specific reason to do the import here?

@karyon what do you think?

Reply:
probably to avoid a cyclic import, we have that in other locations in that file as well.

Reply:
Ah okay, its fine then :)